test/Table_Tests/src/In_Memory/Table_Date_Spec.enso

```diff
@@ -57,14 +57,14 @@ add_specs suite_builder =
         group_builder.specify "should serialise back to input" <|
             expected_text = normalize_lines <|
                 (enso_project.data / "prime_ministers.csv").read_text
-            delimited = data.expected.to_delimited format=(..Delimited "," line_endings=Line_Ending_Style.Unix)
```

Comment:
Should we leave the old test in to ensure backwards compatibility? Assuming that we have that.